### DISCUSSION TITLE
v4.1.0 (Beta)

### DISCUSSION THREAD

**Discussion by osy** [utmapp/UTM#4583] [Releases]
## Highlights
* **QEMU backend updated to v7.1.0**
* (iOS) **TrollStore build (UTM.HV.ipa) now supports JIT**. Additionally, there is now in-app settings.

## Notes
* **iOS 14 and macOS 11.3** are the minimum supported systems for UTM v4.x. Please use UTM v3.x for support down to iOS 11 and macOS 11
* The configuration backend has been massively rewritten. **Please backup all VMs prior to updating as you will not be able to re-open VMs saved by UTM v4 on older versions of UTM if you decide to downgrade.**

## Changes (v4.1.0)
* Updated QEMU to v7.1.0
* Updated ANGLE and virglrenderer to latest commit
* Fixed an issue where user override of machine property settings was not applied properly
* Localization: updated Japanese (thanks @MMP0)
* Localization: updated Spanish (Latin America) (thanks @JezerM)
* (iOS) Added in-app settings popover which synchronizes with the Settings app (useful for TrollStore installs where the Settings app page is inaccessible) (#4446)
* (iOS) TrollStore: support JIT without needing a separate JIT enabler (thanks @PojavLauncherTeam for the idea)

## Issues
Please check the full list on [Github][2] and help report any bug you find that is not listed.

## Installation

* [iOS](https://docs.getutm.app/installation/ios/)
* [macOS](https://docs.getutm.app/installation/macos/)

| File | Description | Installation | JIT | Hypervisor | USB |
|------|------------|--------------|-----|-----------|-----|
| UTM.deb | Jailbroken iOS version | Open in Cydia, dpkg, or Sileo | Yes | Yes(1) | Yes |
| UTM.dmg | macOS version | Mounting and copying UTM.app to /Applications | Yes | Yes | Yes |
| UTM.ipa | Non-jailbroken iOS version (sideloading) | AltStore, etc (see guide) | Yes(2) | No | No |
| UTM.HV.ipa | Non-jailbroken iOS version (TrollStore) | TrollStore | Yes | Yes(1) | Yes |
| UTM.SE.ipa | Non-jailbroken iOS version (sideloading) | AltStore, enterprise signing, etc | No | No | No |

1. Hypervisor on iOS requires an M1 iPad.
2. Enabling JIT may require a separate JIT enabler such as [Jitterbug][3] or Jitstreamer.

[1]: https://getutm.app/install/
[2]: https://github.com/utmapp/UTM/issues
[3]: https://github.com/osy/Jitterbug

<hr /><em>This discussion was created from the release <a href='https://github.com/utmapp/UTM/releases/tag/v4.1.0'>v4.1.0 (Beta)</a>.</em>

**Comment by conath** [comment]
Thanks, the Trollstore version works great on my M1 iPad Pro - my dreams have come true! 😊 

Is it possible to get Trollstore-based JIT working on non-M1 iPad? My A12Z iPad with iOS 14.8 can install the Trollstore version but does not allow me to run emulated VMs.
